specification/splitio/resource-manager/SplitIO.Experimentation/preview/2024-03-01-preview/splitio.json

```diff
@@ -0,0 +1,797 @@
+      "description": "The provisioning state of a resource type.",
+      "enum": [
+        "Succeeded",
+        "Failed",
+        "Canceled"
+      ],
+      "x-ms-enum": {
+        "name": "ResourceProvisioningState",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "Succeeded",
+            "value": "Succeeded",
+            "description": "Resource has been created."
+          },
+          {
+            "name": "Failed",
+            "value": "Failed",
+            "description": "Resource creation failed."
+          },
+          {
+            "name": "Canceled",
+            "value": "Canceled",
+            "description": "Resource creation was canceled."
+          }
+        ]
+      }
+    },
+    "DataSource": {
+      "type": "object",
+      "description": "Data source that telemetry data required for experimentation will be retrieved from.",
+      "properties": {
+        "logAnalytics": {
+          "$ref": "#/definitions/LogAnalytics",
+          "description": "Log Analytics data source."
+        }
+      }
+    },
+    "DataSourceUpdate": {
+      "type": "object",
+      "description": "Data source that telemetry data required for experimentation will be retrieved from.",
+      "properties": {
+        "logAnalytics": {
+          "$ref": "#/definitions/LogAnalyticsUpdate",
+          "description": "Log Analytics data source."
+        }
+      }
+    },
+    "ExchangeTokenParameters": {
+      "type": "object",
+      "description": "The exchange token parameters for exchanging Split.IO data plane authentication token.",
+      "properties": {
+        "token": {
+          "type": "string",
+          "format": "password",
+          "description": "The bearer token that is used for exchanging Split.IO data plane authentication token.",
+          "x-ms-secret": true
+        },
+        "resource": {
+          "type": "string",
+          "description": "The resource of Split.IO data plane authentication token."
+        }
+      },
+      "required": [
+        "token",
+        "resource"
+      ]
+    },
+    "ExchangeTokenResult": {
+      "type": "object",
+      "description": "The exchange token result that contains the bearer token which is used for Split.IO data plane authentication.",
+      "properties": {
+        "token": {
+          "type": "string",
+          "format": "password",
+          "description": "The bearer token that is used for Split.IO data plane authentication.",
+          "readOnly": true,
+          "x-ms-secret": true
+        }
+      },
+      "required": [
+        "token"
+      ]
+    },
+    "ExperimentationWorkspace": {
+      "type": "object",
+      "description": "A ExperimentationWorkspace Resource by Split.IO",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ExperimentationWorkspaceProperties",
+          "description": "The resource-specific properties for this resource.",
```

Comment:
>The resource-specific properties for this resource [](http://example.com/codeflow?start=26&length=50)

nit: May be a better description ?

Reply:
This description is not controlled by us. It get auto-generated based on TypeSpec when using `model ExperimentationWorkspace
  is TrackedResource<ExperimentationWorkspaceProperties>`